### PR TITLE
Update EntityDamageEvent.php

### PR DESCRIPTION
**Pull request by tigerza117**

### CODE REVIEW

File: src/pocketmine/event/entity/EntityDamageEvent.php [hunk 1]
```diff
@@ -107,75 +108,77 @@ public function __construct(Entity $entity, $cause, $damage){
...
-				case self::CAUSE_FIRE:
-				case self::CAUSE_LAVA:
-				case self::CAUSE_BLOCK_EXPLOSION:
-				case self::CAUSE_ENTITY_EXPLOSION:
-				case self::CAUSE_LIGHTNING:
-					$points = 0;
-					foreach($entity->getInventory()->getArmorContents() as  $i){
-						if($i->isArmor()){
-							$points += $i->getArmorValue();
+			if($entity->getInventory() instanceof PlayerInventory){
```
Comment (ishitatsuyuki) [inline src/pocketmine/event/entity/EntityDamageEvent.php]: Why? Player without PlayerInventory?

**Comment by dog194** [comment]
Closing for formatter. I will change this later. And thanks for making this.
BTW, which plugins make Player entity donot have PlayerInventory

**Comment by tigerza117** [comment]
Fix this $entity->getInventory()->getArmorContents()  on null error

File: src/pocketmine/event/entity/EntityDamageEvent.php (hunk 1 above)
Comment (tigerza117) [inline src/pocketmine/event/entity/EntityDamageEvent.php]: Fix this $entity->getInventory()->getArmorContents() on null error

**Comment by Driesboy** [comment]
its a big bug!

**Pull request closed without merging**